umm-spec-lib/src/cmr/umm_spec/location_keywords.clj

```diff
@@ -43,20 +43,15 @@
 (defn- location-values
   "Returns the location keyword values in order so that we can get the last one"
   [location-keyword]
-  (for [k location-keyword-order
```

Comment:
Now that it no longer makes use of the location-keyword-order:
1. Should the doc string be modified? 
2. location-keyword-order def should be removed from the file since no where else is using it.